### ISSUE TITLE
Error on SingleLinePerSelector linter

### ISSUE THREAD

**Issue by Anahkiasen** [brigade/scss-lint#199]
Using 0.26

```
SCSSLint::Linter::SingleLinePerSelector raised unexpected error linting file public/app/sass/modules/_forms.scss: 'undefined method `gsub' for nil:NilClass'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter/single_line_per_selector.rb:36:in `normalize_spacing'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter/single_line_per_selector.rb:22:in `invalid_comma_placement?'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter/single_line_per_selector.rb:7:in `visit_rule'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/sass-3.3.11/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:124:in `visit'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:134:in `block in visit_children'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:132:in `each'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:132:in `visit_children'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/sass-3.3.11/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter/single_line_per_selector.rb:8:in `visit_rule'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/sass-3.3.11/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:124:in `visit'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:134:in `block in visit_children'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:132:in `each'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:132:in `visit_children'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/sass-3.3.11/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:124:in `visit'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/linter.rb:20:in `run'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/runner.rb:58:in `run_linter'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/runner.rb:42:in `block in find_lints'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/runner.rb:37:in `each'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/runner.rb:37:in `find_lints'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/runner.rb:21:in `block in run'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/runner.rb:20:in `each'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/runner.rb:20:in `run'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/lib/scss_lint/cli.rb:99:in `run'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/bin/scss-lint:8:in `block in <top (required)>'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/bin/scss-lint:6:in `tap'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/gems/scss-lint-0.26.0/bin/scss-lint:6:in `<top (required)>'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/bin/scss-lint:23:in `load'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/bin/scss-lint:23:in `<main>'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `eval'
/Users/anahkiasen/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `<main>'
```

This is the file containing a variable in its selector (see #190) so may be related.

``` yaml
  SingleLinePerSelector:
    enabled: true
```
